styles.css

```diff
@@ -0,0 +1,18 @@
+body {
+    background-color: white;
+}
+
+.ejemplo{
```

Comment:
when you remove the figure tag you could add this class to the section element and have the styles in the section